src/plugins/data_views/server/rest_api_routes/public/has_user_data_view.ts

```diff
@@ -46,6 +46,11 @@ const hasUserDataViewRouteFactory =
     router.versioned.get({ path, access: 'internal' }).addVersion(
       {
         version: '1',
+        security: {
+          authz: {
+            requiredPrivileges: ['indexPatterns:read'],
```

Comment:
Do you know what this endpoint is actually used for? I'm not sure.

Reply:
Looks like it's used to determine which "no data" state page we show - the one to add data to ES or to create a data view.